src/main/java/net/clgd/ccemux/emulation/CCEmuX.java

```diff
@@ -194,7 +214,8 @@ public int assignNewID() {
 	@Override
 	public IMount createResourceMount(String domain, String subPath) {
 		String path = Paths.get("assets", domain, subPath).toString().replace('\\', '/');
-		if (path.startsWith("\\")) path = path.substring(1);
+		if (path.startsWith("\\"))
```

Comment:
Nooooo. This is how you get bugs. Multi-line if statements should always have braces